backend/src/validators/user.ts

```diff
@@ -4,7 +4,7 @@ import { body } from "express-validator";
 // https://express-validator.github.io/docs/guides/validation-chain
 // https://github.com/validatorjs/validator.js#validators
 
-const nameValidator = body("name")
+const nameValidator = body("firstName")
```

Comment:
Do we need a validator for lastName as well?

Reply:
Oh good catch, that would probably be a good idea, just updated. Thanks!

Reply:
Thanks!